app/plugin_install.go

```diff
@@ -61,40 +71,45 @@ func (a *App) installPlugin(pluginFile io.ReadSeeker, replace bool) (*model.Mani
 	// Store bundle in the file store to allow access from other servers.
 	pluginFile.Seek(0, 0)
 
-	if _, err := a.WriteFile(pluginFile, a.getBundleStorePath(manifest.Id)); err != nil {
-		return nil, model.NewAppError("uploadPlugin", "app.plugin.store_bundle.app_error", nil, err.Error(), http.StatusInternalServerError)
+	if _, appErr := a.WriteFile(pluginFile, a.getBundleStorePath(manifest.Id)); appErr != nil {
+		return nil, model.NewAppError("uploadPlugin", "app.plugin.store_bundle.app_error", nil, appErr.Error(), http.StatusInternalServerError)
 	}
 
+	// Notify cluster peers async.
```

Comment:
Is this comment adding value?

Reply:
since we have both async (`SendClusterMessage`) and sync (`ConfigChanged`), versions, thought it'd would help to clarify when writing code not to depend on the operation being sync.

Reply:
I admit the comment doesn't really help capture the nuance of those two methods -- that is, nothing here tells me about `ConfigChanged` and why I care that this is async vs. that being sync.